kolibri/plugins/facility/assets/src/views/UserCreatePage.vue

```diff
@@ -185,10 +186,29 @@
         ];
       },
     },
+    watch: {
+      kind(val) {
+        this.showPasswordInput =
+          val.value !== UserKinds.LEARNER ||
+          !this.facilityConfig.learner_can_login_with_no_password;
+        this.updatePasswordValues();
+      },
+    },
     mounted() {
       this.$store.dispatch('notLoading');
+      this.getFacilityConfig(this.currentFacilityId).then(() => {
+        this.showPasswordInput = !this.facilityConfig.learner_can_login_with_no_password;
+        this.updatePasswordValues();
+      });
     },
     methods: {
+      ...mapActions(['getFacilityConfig']),
+      updatePasswordValues() {
+        if (!this.showPasswordInput) {
+          this.passwordValid = true;
+          if (this.password === '') this.password = 'NOT_SPECIFIED';
```

Comment:
will this work as expected? I would imagine if we're setting this string client-side, it's going to get hashed and salted by Django which is not the intended behavior

Reply:
no, if you look at the backend code changes, when it receives `NOT_SPECIFIED` as password, it's not hashed, but written in plain text in the db